examples/tutorial.ipynb

```diff
@@ -355,7 +355,7 @@
     "    return x + (1 + 3*x**2)/(1 + x + x**3)\n",
     "\n",
     "\n",
-    "def dG_ddYdx(x, y, dY_dx):\n",
+    "def dG_ddYdx(x, Y, dY_dx):\n",
```

Comment:
The earlier Notebook sections seemed to prefer using `Y` instead of `y`.

`simple_01`, `lagaris_01`, etc. use the `y` variable in this position as well, while `example_pde1ivp_01` uses `Y` in this position.

These might be adjusted for more consistent style, but it shouldn't affect results.

Reply:
Thanks for catching that. The convention we use is for Y as the equation solution, so your correction is needed.